apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -73,11 +76,30 @@ const trackBlockReplacement = ( originalBlockIds, blocks ) => {
 	} );
 };
 
+/**
+ * Track inner blocks replacement.
+ * This is how Page Templates insert their content into page, by replacing everything that was already there.
+ *
+ * @param {Array} rootClientId id of parent block
+ * @param {object|Array} blocks block instance object or an array of such objects
+ * @returns {void}
+ */
+const trackInnerBlocksReplacement = ( rootClientId, blocks ) => {
+	castArray( blocks ).forEach( block => {
+		tracksRecordEvent( 'wpcom_block_inserted', {
+			block_name: block.name,
+			blocks_replaced: false,
+			// isInsertingPageTemplate filter is set by Starter Page Templates
+			from_template_selector: applyFilters( 'isInsertingPageTemplate', false ),
```

Comment:
Wondering if does it worth defining a global constants file in order to share them across these plugins.